### ISSUE TITLE
validate all html/xml/svg/xslt files

### ISSUE THREAD

**Issue by kellerfisch** [StudentsAgainstCovid19/corona-aid#16]
## Html
- [x] index.html
- [x] colors.html
- [x] navigation.html
- [x] optimization.html
- [x] piecharts.html
- [x] priority.html

### Examples
- [x] infected_compressed_example.xml
- [x] infected_id_example.xml
- [x] config_example.xml
- [x] symptoms_example.xml

### New DTDs
- [x] help_popup.dtd --> validation of help_popup.xml necessary after merge into master (!)
- [x] district.dtd
- [x] district_analytics.dtd

### DTDs that were not yet linked in the JS code (temporarily constructed XML files, see the example XML files)
- [x] calculate_prio_result.xml
- [x] construct_symptom_popup_xml_result.xml
- [x] create_pie_chart_result.xml
- [x] parse_symptoms_result.xml
- [x] push_history_item.xml
- [x] push_prescribe_test.xml
~realtime_sse.xml~
~realtime_update_merged.xml~

## XSLT
All XSLT documents were already perfectly valid 😎

- [x] xslt_calculate_prio.xsl
- [x] xslt_call_list.xsl
- [x] xslt_clustered_list.xsl
- [x] xslt_construct_help.xsl
- [x] xslt_detailed_view.xsl
- [x] xslt_edit_symptoms.xsl
- [x] xslt_merge_symptoms.xsl
- [x] xslt_notes_popup.xsl
- [x] xslt_parse_symptoms.xsl
- [x] xslt_pie_chart.xsl
- [x] xslt_prio_gpx.xsl
- [x] xslt_progressbar.xsl
- [x] xslt_realtime_update.xsl
- [x] xslt_show_districts.xsl
- [x] xslt_show_illnesses.xsl
- [x] xslt_symptom_div.xsl

### assets/icons
- [x] blur_filter.svg
- [x] checkbox_not_set.svg
- [x] check-circle.svg
- [x] close_button.svg
- [x] exampleWellbeingHistory.svg
- [x] help.svg
- [x] last_button.svg
- [x] list_button.svg
- [x] list_button2.svg
- [x] next_button.svg
- [x] reset_zoom.svg
- [x] search_magnifier.svg
- [x] wellbeing_slider.svg

### assets/markers
- [x] example_pie_chart.svg
- [x] high_prio.svg
- [x] intermed_prio.svg
- [x] lower_prio.svg
- [x] tried_call.svg
- [x] veryhigh_prio.svg

### assets/wellbeing_indicators
- [x] wellbeing_bad.svg
- [x] wellbeing_good.svg
- [x] wellbeing_intermediate.svg
- [x] wellbeing_verybad.svg
- [x] wellbeing_verygood.svg

**Comment by kellerfisch** [comment]
Current status @Ordinateur-Hack ?

**Comment by marcauberer** [comment]
There is a pr open for that, please do not merge before we have the confirmation from @Ordinateur-Hack

**Comment by kellerfisch** [comment]
Current status @Ordinateur-Hack ?

**Comment by Splines** [comment]
@keksklauer4 Thanks for the great overview. I will finish this today 😅

**Comment by marcauberer** [comment]
@Ordinateur-Hack: @keksklauer4  removed some files while restructuring the code. Those files do not exist any more. I'm going to remove them from the list above. Your pr is merged btw.
